### ISSUE TITLE
Cache policy warning

### ISSUE THREAD

**Issue by erlapso** [GoogleChrome/lighthouse#6627]
The performance check (PageSpeed test I’m looking at you) is reporting that 23 resources have an inefficient cache policy, but the detail actually shows that the cache policy is over 2 days, sometimes over 30 days

**Comment by wardpeet** [comment]
@patrickhulce can you answer this one? I can't make sense of the [uses-long-cache-ttl audit](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/byte-efficiency/uses-long-cache-ttl.js), it looks like we do some magic to show these results (calculate cache probability, ...)

![image](https://user-images.githubusercontent.com/1120926/48872872-59884380-edeb-11e8-86d6-49d00f498ba4.png)

**Comment by patrickhulce** [comment]
If it's anything less than ~6 months then it will show up here. Our recommendation is a year because if you're OK with having a 29 day old version is there really that much of a difference from a multi-month old version? If you cared about having it up-to-date you should probably be using must-revalidate :)

**Comment by robdodson** [comment]
That's good to know. I don't think the guide that LH currently links to explains what the cache policy would need to be in order to pass the audit. If we're writing updated guidance on web.dev we can explain that.

**Comment by Malvoz** [comment]
> If we're writing updated guidance on web.dev [...]

> If you cared about having it up-to-date you should probably be using must-revalidate

[`must-revalidate`](https://tools.ietf.org/html/rfc7234#section-5.2.2.1) is only taking effect after the resource has become stale (e.g. `max-age` has expired), instead use [`no-cache`](https://tools.ietf.org/html/rfc7234#section-5.2.2.2) which: "_indicates that the response MUST NOT be used to satisfy a subsequent request without successful validation on the origin server._"

**Comment by patrickhulce** [comment]
Thanks @Malvoz! I was referring to the fact that if you had something that really must be cached for 30 days but not any longer, you should be using `must-revalidate`. We use both `no-cache` and `must-revalidate` as signals for filtering. We won't flag assets with either.

**Comment by robdodson** [comment]
@meggin fyi as we work on updating our audit docs :)

**Comment by Splaktar** [comment]
> That's good to know. I don't think the guide that LH currently links to explains what the cache policy would need to be in order to pass the audit. If we're writing updated guidance on web.dev we can explain that.

Indeed, it would be nice for https://web.dev/uses-long-cache-ttl/ to state that it needs to be 6+ months to pass this audit.
